src/device/clint.rs

```diff
@@ -54,8 +51,6 @@ impl TimestampEntry {
 /// Represents a virtual CLINT (Core Local Interruptor) device
 #[derive(Debug)]
 pub struct VirtClint {
-    /// A driver for the physical CLINT
-    driver: &'static Mutex<ClintDriver>,
```

Comment:
We do want to keep a driver in the virt clint. There might be multiple CLINT (e.g. on NUMA systems).
What we want to do is remove the lock.